src/assets/sass/_variables.scss

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@500&display=swap');
+
+// COLORS
+$primary-color: #B4DDC0;
```

Comment:
I am not sure about these names, perhaps something more descriptive?

```scss
$primary-color-green : #B4DDC0;
```
Perhaps we group them similar to how Jake has on the design system page.

![image](https://user-images.githubusercontent.com/55165280/175986672-62a57a61-b6c9-42ca-8b86-b22ea2a5e487.png)